src/file/SDL_iostream.c

```diff
@@ -558,6 +558,160 @@ SDL_IOStream *SDL_IOFromFP(FILE *fp, bool autoclose)
 }
 #endif // !HAVE_STDIO_H && !defined(SDL_PLATFORM_WINDOWS)
 
+#if !defined(SDL_PLATFORM_WINDOWS)
+
+// Functions to read/write file descriptors. Not used for windows.
+
+#ifdef HAVE_LSEEK64
+#define lseek_off_t off64_t
+#define lseek       lseek64
+#else
+#define lseek_off_t off_t
+#endif
```

Comment:
This part shouldn't actually be necessary, at least on glibc, because CMakeLists.txt sets `_FILE_OFFSET_BITS=64`, which turns ordinary `lseek()` and `off_t` into aliases for `lseek64()` and `off64_t`.

(But maybe it's necessary on some other Unix?)